libs/sdk-ui-dashboard/src/presentation/widget/insight/ViewModeDashboardInsight/useInsightPositionStyle.ts

```diff
@@ -0,0 +1,34 @@
+// (C) 2025 GoodData Corporation
+
+import { CSSProperties, useMemo } from "react";
+import { insightVisualizationUrl, IInsight } from "@gooddata/sdk-model";
+
+import { useDashboardSelector, selectSettings } from "../../../../model/index.js";
+import { DASHBOARD_LAYOUT_RESPONSIVE_SMALL_WIDTH } from "../../../constants/index.js";
+
+export const useInsightPositionStyle = (insight: IInsight, clientWidth?: number) => {
+    const { enableKDWidgetCustomHeight, enableDashboardFlexibleLayout } =
+        useDashboardSelector(selectSettings);
+
+    const isPositionRelative =
+        insight &&
+        insightVisualizationUrl(insight).includes("headline") &&
```

Comment:
you can use insightVisualizationType(insight) directly

Reply:
Fixed. I did not know about this helper and just moved the old code to the new location with slight tweeks.